content/updates/tf-terminology-breakdown.md

```diff
@@ -0,0 +1,210 @@
+---
+visible: true
+draft: false
+title: Terraform + OpenTofu Terminology Breakdown
+slug: terraform-opentofu-terminology-breakdown
+author: Matt Gowie
+date: 2025-03-01
+description: List of Terraform and OpenTofu terms with definitions and explanations.
+image: /img/updates/terraform-null-label-part1.png
+callout: <p>👋 <b>Got a term that you're still confused on in the TF or IaC space that you want us to include here? <a href='/contact'>Get in touch and we'd be happy to add it!</a>
+---
+
+# Table of Contents
+
+- [Table of Contents](#table-of-contents)
```

Comment:
I'd vote for removing this line - the reader knows there's a table of contents because that's where they're reading

Reply:
Ha this was actually a pain because the tool I was using kept adding it back. Done in latest updates though.